src/components/postgresConnect.ts

```diff
@@ -678,7 +678,7 @@ class PostgresDB {
   public getReviewStageAndLevel = async (reviewId: number) => {
     const text = `
       SELECT review.level_number AS "levelNumber",
-      stage_number as "stageNumber"
+      review.stage_number as "stageNumber"
       FROM review JOIN review_assignment ra
       ON review.review_assignment_id = ra.id
```

Comment:
Since you've now included `stage_number` in `review`, this query no longer needs to JOIN with review_assignment.

Reply:
true: https://github.com/openmsupply/application-manager-server/pull/318/commits/a511f567c430c0d5a6524881fb6742e57ea346cb